src/components/NcAppSettingsDialog/NcAppSettingsDialog.vue

```diff
@@ -146,16 +146,13 @@ export default {
 		<template v-if="hasNavigation" #navigation="{ isCollapsed }">
 			<ul v-if="!isCollapsed"
 				:aria-label="settingsNavigationAriaLabel"
-				class="navigation-list"
-				role="tablist">
+				class="navigation-list">
 				<li v-for="section in sections" :key="section.id">
-					<a :aria-selected="section.id === selectedSection"
-						:class="{
+					<a :class="{
```

Comment:
```suggestion
					<a :aria-current="section.id === selectedSection"
						:class="{
```

Should we not keep the information of the current active one?

Reply:
This attribute was not correct anyway 🙈
It should have been a string, not boolean in old Vue.

Reply:
But I wonder if this should be navigation and links at all. It is not related to the actual navigation, there is no `href`.

But it has kind of navigation semantic.

I dunno

Reply:
The aria-selected attribute indicates the current "selected" state for gridcell, option, row and tab roles.
> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected

We have no tab roles anymore

Reply:
> links at all. It is not related to the actual navigation, there is no href.

oohhh, good point. i have to replace it too

Reply:
>  there is no href

I would say keep links, but add the href as `#${section.id}`.

> The aria-selected attribute indicates the current "selected" state for gridcell, option, row and tab roles.

Yes thats why I suggested `aria-current` :)

Reply:
or `? 'true' : 'false'`

Reply:
> With any aria-* it should be `(section.id === selectedSection).toString()`

Or short form `${section.id === selectedSection}`?

Reply:
> > there is no href
> 
> I would say keep links, but add the href as `#${section.id}`.

If we add anchor links then it wouldn't it be less accessible as the anchor at the end of the URL makes it seem like opening the same URL auto-opens the settings dialog to the same nav section when it doesn't

There is currently no special URL param to directly open the setting dialog either 🤔

Reply:
> Or short form? (Not sure about readability)

Too many options, let's just migrate to Vue 3 finally =D

Reply:
@susnux it should not be a link as @Pytal already said. Could you please explain your point why do we need `aria-current` for navigation items?

Reply:
I think that it is best to make it a link. I agree with @Pytal thoughts, but it doesn't seem to me critical.

This list:
- is actually navigation (on the page). The only action click does is scrolling to the element with a specific ID
- has an item with a "current" state with also changes during scrolling

So semantically, it is very close to navigation with links from my PoV, rather than a button.

The only problem is that the dialog is not reopened on page reload with opened dialog... Do you think this is a serious issue? 

---

The best would be, it would also fix current keyboard navigation:
1. A user presses on button 3 "WebDAV" in navigation,
2. It scrolls to the section 3
3. A user presses tab or uses a screen reader
   - **Expected**: the focus is moved to the next element in WebDAV section
   - **Actual:** the focus is moved to the next navigation button

And this behavior is also very "link" behaviour.
---

But I'm also fine with buttons if we move the focus.